### DISCUSSION TITLE
Karavan atlasmap integration sources are available???

### DISCUSSION THREAD

**Discussion by kondetiramanakumar** [apache/camel-karavan#1186] [Q&A] [answered]
Karavan atlasmap integration sources are available???

**Comment by davsclaus** [comment, marked as answer]
atlasmap is EOL and not maintained anymore. There is no atlasmap support in Karavan or Apache Camel anymore.

**Comment by kondetiramanakumar** [comment]
@davsclaus 

please help me out While building the karavan designer I am getting the  below Issue.

<img width="917" alt="image" src="https://github.com/apache/camel-karavan/assets/14121796/e28180e1-0e10-4d7c-8f2e-eac9c30ffc74">

**Comment by kondetiramanakumar** [comment]
@davsclaus any update on my above request error present in the added screen shot..........

**Comment by kondetiramanakumar** [comment]
@davsclaus  one more issue is module not found while building up the application

![image](https://github.com/apache/camel-karavan/assets/14121796/e6718523-7b5d-4c5d-be91-400058c9eaf8)

**Comment by davsclaus** [comment]
this discussion is about atlasmap, not about how to build the source code. You need to open a new discussion for that.
